pulp_file/tests/functional/api/test_crud_remotes.py

```diff
@@ -16,7 +16,6 @@
     FILE_REMOTE_PATH
 )
 from pulp_file.tests.functional.utils import gen_file_remote, skip_if
-from pulp_file.tests.functional.utils import set_up_module as setUpModule  # noqa:F401
```

Comment:
Why are we removing this?

Reply:
The linter picked up that setUpModule was not used. I didn't see a use case for it within the file. 

Is there a usage of this that I am missing? 

All tests passed with the removal of the line.

```
(pulp3) [herring@redherring api]$ pylint test_crud_remotes.py 
************* Module pulp_file.tests.functional.api.test_crud_remotes
test_crud_remotes.py:19:0: W0611: Unused set_up_module imported from pulp_file.tests.functional.utils as setUpModule (unused-import)

-----------------------------------
Your code has been rated at 9.90/10

(pulp3) [herring@redherring api]$
```

Reply:
Had a discussion about this. Understood this is now part of the module setup.

Re-adding